libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -688,7 +688,27 @@ void SITL_State::_simulator_servos(struct sitl_input &input)
     }
 
     if (_vehicle == ArduPlane) {
-        throttle = constrain_float((input.servos[2] - 1000) / 1000.0f, 0.0f, 1.0f);
+        float forward_throttle = constrain_float((input.servos[2] - 1000) / 1000.0f, 0.0f, 1.0f);
+        // do a little quadplane dance
+        float hover_throttle = 0.0f;
+        uint8_t running_motors = 0;
+        for (i=0; i<4; i++) {
```

Comment:
Why 4?  We have 8 motors on our quadplanes :-P

This looks like the Copter case ought to be factored out (rather than copied?).

Reply:
Is there a way of telling how many motors are configured? Copter uses channels but we are using some of those for the forward thrust motor.

Reply:
It's also not the same as copter - note the input.servos[i+4]

Reply:
I tried sitl_model->num_motors but that's protected and the motor_offset is buried deep inside the quadplane frame - a steer on where these should come from would be appreciated!